### DISCUSSION TITLE
Dual monitor

### DISCUSSION THREAD

**Discussion by sebastichoupinet** [hyprwm/Hyprland#190] [Q&A] [answered]
Hi

First thanks **a lot**  for such an amazing work!
It's so smooth and neat, I'm very happy to use it.

Just a quick question: how can we configure dual monitors ?

I've tried with that , it works, but I can move the mouse from one screen to another (but using the binding works).

Also, I don't know why the VGA-A-1, even if set to the nineth workspace, keeps apparing on the workspace 2

`monitor=HDMI-A-1,1920x1080@144,0x0,1
workspace=HDMI-A-1,1

monitor=VGA-1,1920x1080@144,0x1921,1
workspace=VGA-A-1,9`

If it's  a lack of understanding of the documentation, could you please give the name of the section please ?

In any case, thanks again

**Comment by fufexan** [comment, marked as answer]
> VGA-A-1

You've defined it as `VGA-1`. That's why it doesn't work.

> `0x1921`

It is `XxY` so you probably want `1920x0`. The 1px difference isn't needed, and would probably cause the cursor not to travel between monitors.

**Reply by sebastichoupinet** [reply]
Ah thanks a lot , it works
I was stupid to confound x and y coordinates
And effectively it works


For the posterity (if anyone needs somethings similar) 

`
monitor=HDMI-A-1,1920x1080@144,0x0,1
workspace=HDMI-A-1,1

monitor=VGA-1,1920x1080@144,1920x0,1
workspace=VGA-1,9`

**Comment by rafay99-epic** [comment]
Hey, Can you help me out? I have a connected display using HDMI via a Type C. That means type C is connected to my computer, and the HDMI is connected to the type C dock.

I can't seem to identify my monitor number or configure my Hyperion config file.
My display is a HP Elite Display E-240. 

I have tried multiple configuration but it does seem to work any suggestions.
